pineappl_cli/build.rs

```diff
@@ -0,0 +1,62 @@
+#[cfg(feature = "fastnlo")]
+fn main() {
+    use std::process::Command;
+
+    let mut bridge = cxx_build::bridge("src/import.rs");
+
+    // PineAPPL CAPI
+    let pineappl_capi = pkg_config::Config::new()
+        .atleast_version("0.5.0")
+        .probe("pineappl_capi")
+        .unwrap();
+
+    for include_path in pineappl_capi.include_paths {
+        bridge.include(include_path);
+    }
+
+    for lib_path in pineappl_capi.link_paths {
+        println!("cargo:rustc-link-search={}", lib_path.to_str().unwrap());
+    }
+
+    for lib in pineappl_capi.libs {
+        println!("cargo:rustc-link-lib={}", lib);
+    }
+
+    // fastNLO
+    let fnlo_include_path = String::from_utf8(
+        Command::new("fnlo-tk-config")
+            .arg("--incdir")
+            .output()
+            .unwrap()
+            .stdout,
+    )
+    .unwrap();
+
+    bridge.include(fnlo_include_path);
+
+    let fnlo_lib_path = String::from_utf8(
+        Command::new("fnlo-tk-config")
+            .arg("--libdir")
+            .output()
+            .unwrap()
+            .stdout,
+    )
+    .unwrap();
+
+    println!("cargo:rustc-link-search={}", fnlo_lib_path);
+
+    // TODO: why do I have to link statically?
+    println!("cargo:rustc-link-lib=static=fastnlotoolkit");
+    println!("cargo:rustc-link-lib=z");
+
+    // bridging code
+    bridge.file("src/import/fastnlo.cpp").compile("cxx-bridge");
+
+    println!("cargo:rerun-if-changed=src/import.rs");
+    println!("cargo:rerun-if-changed=src/import/import-ffirs");
+    println!("cargo:rerun-if-changed=src/import/fastnlo.cpp");
+    println!("cargo:rerun-if-changed=src/import/fastnlo.hpp");
+}
+
+#[cfg(not(feature = "fastnlo"))]
+fn main() {}
```

Comment:
Two options for 

1. if you like to do what I proposed, and having a separate crate for "fastNLO bindings", than we need to do nothings right now (`build.rs` will be simply the build file of that crate, and we'll drop the feature gates, the only option will be spelled out in `pineappl_cli/Cargo.toml`)
2. if we're going to keep everything in PineAPPL CLI, then you'll need a separate function for the time APPLgrid will join the party (and the feature gate we'll be placed not on `main()` but on `fastNLO()`)

Personally, I still prefer option 1..